it/lessons/basics/enum.md

```diff
@@ -86,6 +122,13 @@ iex> Enum.min([5, 3, 0, -1])
 -1
 ```
 
+`min/2` fornische le stesse funzionalità di `min/1`, ma in caso di enumerazione vuota, permette di specificare una funzione per produrre il valore minimo 
```

Comment:
Typo here: `fornische` should be `fornisce`.

Reply:
fixed in efe71e5a7017e4a5602d069b368c9c56d9880e4b